flight/Modules/Logging/logging.c

```diff
@@ -285,6 +352,81 @@ static void loggingTask(void *parameters)
 	}
 }
 
+
+/**
+ * Log all settings objects
+ * \param[in] obj Object to log
+ */
+static void logSettings(UAVObjHandle obj)
+{
+	if (UAVObjIsSettings(obj)) {
+		UAVTalkSendObjectTimestamped(uavTalkCon, obj, 0, false, 0);
+	}
+}
+
+/**
+ * Write log file header
+ * see firmwareinfotemplate.c
+ */
+static void writeHeader()
+{
+	int pos;
+	char tmp_str[45];
+	char *info_str;
+	char this_char;
+	DateTimeT date_time;
+
+	const struct pios_board_info * bdinfo = &pios_board_info_blob;
+
+	sprintf(tmp_str, "%s\n", "Tau Labs git hash:");
+	send_data((uint8_t*)tmp_str, strlen(tmp_str));
```

Comment:
Why not directly send the constant?  #define LOG_HEADER "Tau Labs git hash:\n"
send_data(LOG_HEADER, strlen(LOG_HEADER)) will evaluate the strlen nicely at compile time etc.